components/script/dom/bindings/codegen/CodegenRust.py

```diff
@@ -5324,14 +5324,14 @@ def __init__(self, config, prefix, webIDLFile):
 
         # Do codegen for all the typdefs
         for t in typedefs:
-            if t.innerType.isUnion():
-                cgthings.extend([CGGeneric("\npub type %s = %s;\n\n" % (t.identifier.name,
-                                                                        "UnionTypes::" + str(t.innerType)))])
-            else:
-                assert not typeNeedsRooting(t.innerType, config.getDescriptorProvider)
-                cgthings.extend([CGGeneric("\npub type %s = " % (t.identifier.name)),
-                                 getRetvalDeclarationForType(t.innerType, config.getDescriptorProvider()),
-                                 CGGeneric(";\n\n")])
+            for e in enums:
```

Comment:
I don't think this is limited to just enums. We want all typedefs to be re-exported.